reproducibility_project/src/engines/gomc/project.py

```diff
@@ -1,23 +1,2159 @@
+# please just enter and empty string (i.e., "" or '')
+gomc_binary_path = "/Users/brad/Programs/GOMC/GOMC_dev_8_25_21/bin"
+
+# number of MC cycles
+MC_cycles_melt_equilb_NVT = 5 * 10 ** 3  # set value for paper = 5 * 10 ** 3
+MC_cycles_equilb_NVT = 5 * 10 ** 3  # set value for paper = 5 * 10 ** 3
+MC_cycles_equilb_design_ensemble = (
+    40 * 10 ** 3
+)  # set value for paper = 40 * 10 ** 3
+MC_cycles_production = 120 * 10 ** 3  # set value for paper = 120 * 10 ** 3
+
+output_data_every_X_MC_cycles = 10
+
+# max number of equilibrium selected runs
+equilb_design_ensemble_max_number = 3
+
+# force field (FF) file for all simulations in that job
+# Note: do not add extensions
+ff_filename_str = "in_FF"
+
+# initial mosdef structure and coordinates
+# Note: do not add extensions
+mosdef_structure_box_0_name_str = "mosdef_box_0"
+mosdef_structure_box_1_name_str = "mosdef_box_1"
+
+# melt equilb simulation runs GOMC control file input and simulation outputs
+# Note: do not add extensions
+melt_equilb_NVT_control_file_name_str = "melt_NVT"
+
+# equilb simulation runs GOMC control file input and simulation outputs
+# Note: do not add extensions
+equilb_NVT_control_file_name_str = "equilb_NVT"
+
+# The equilb using the ensemble used for the simulation design, which
+# includes the simulation runs GOMC control file input and simulation outputs
+# Note: do not add extensions
+equilb_design_ensemble_control_file_name_str = "equilb_design_ensemble"
+
+# The production run using the ensemble used for the simulation design, which
+# includes the simulation runs GOMC control file input and simulation outputs
+# Note: do not add extensions
+production_control_file_name_str = "production_run"
+
+
+path_from_job_to_box_inputs = "../../"
+
+walltime_mosdef_hr = 4
+walltime_gomc_hr = 368
+memory_needed = 16
+
+use_pymbar = True  # True of False
+
+ff_info_dict = {
+    "trappe-ua": {
+        "ngpu": 0,
+        "ncpu": 2,
+        "Ewald": False,
+        "ElectroStatic": False,
+        "VDWGeometricSigma": False,
+    },
+    "benzene-ua": {
+        "ngpu": 0,
+        "ncpu": 2,
+        "Ewald": False,
+        "ElectroStatic": False,
+        "VDWGeometricSigma": False,
+    },
+    "spce": {
+        "ngpu": 1,
+        "ncpu": 4,
+        "Ewald": True,
+        "ElectroStatic": True,
+        "VDWGeometricSigma": False,
+    },
+    "oplsaa": {
+        "ngpu": 1,
+        "ncpu": 4,
+        "Ewald": True,
+        "ElectroStatic": True,
+        "VDWGeometricSigma": True,
+    },
+}
+
+# ******************************************************
+# users typical variables, but not all (end)
+# ******************************************************
+
+
+# ******************************************************
+# signac and GOMC-MOSDEF code (start)
+# ******************************************************
+
+# define project
+project = signac.init_project("mosdef_reproducibility")
```

Comment:
Sorry, what i was referencing in my previous review was this line here. This is unnecessary, and you should keep the `if __name__ == "__main__"` section below.
```suggestion
```

Reply:
changed as requested